### DISCUSSION TITLE
PeoplePicker.SelectedPeople JSON parsing error

### DISCUSSION THREAD

**Discussion by Havgo** [microsoft/powercat-creator-kit#376] [Q&A] [answered]
When a PeoplePicker component is blank (on app start, or after clearing selection), any reference to it (PeoplePicker.SelectedPeople) returns an error "JSON Parsing Error, Expected 'Array' But Got 'Undefined'"

Unlike selecting an empty collection, which would Display "We didn't find any data", Selecting PeoplePicker.SelectedPeople displays "Error".

For reference:

Suggestions_Items: 
TBcolMeetingPeopletemp

OnSearch:
ClearCollect(
    TBcolMeetingPeopletemp,
        Search(
            Filter(colHRdata_EmployeeRegistry,!IsBlank(WorkEmail)),Self.SearchText,"FullName")
)

Gallery with Items: 
PeoplePicker.SelectedPeople

Returns the error in the screenshot

![Screenshot 2023-03-22 165309](https://user-images.githubusercontent.com/128643380/227035515-9491eeb0-9a64-44a5-9c22-f2b4ca733a1b.png)

**Comment by dineshary** [comment]
I am facing the same error. Is there a workaround for this issue?

**Comment by WinOnMarco** [comment]
Same here. Any ideas?

**Comment by denise-msft** [comment, marked as answer]
Please try branching to a blank value instead of the PeoplePicker property if there is a blank or error state using following formula in the Items property of the gallery or referring component: 
`If(!IsBlankOrError(DataCardValue6_1.SelectedPeople),DataCardValue6_1.SelectedPeople, Blank())`

**Comment by denise-msft** [comment]
@Havgo @WinOnMarco @dineshary please let me know whether the workaround works for you. Thank You.

**Comment by benjy-evans** [comment]
The workaround suggested worked for me when using the PeoplePicker. Is there any intent to resolve this bug, beyond the workaround?
